### DISCUSSION TITLE
Git Hub copailot

### DISCUSSION THREAD

**Discussion by petani97** [community/community#8199] [Copilot Conversations] [answered]
GitHub Copilot could not connect twhy GitHub Copilot can't connect to server. Extension activation failed: "User is not authorized" is there any solution???o server. Extension activation failed: "User not authorized"

**Comment by davecheney** [comment, marked as answer]
I am sorry, you are not currently invited to use Copilot. We are issuing new invites as we have capacity. Thank you for your understanding.

**Reply by petani97** [reply]
do you can help me master to github copailot
